users/templates/users/privacy.html

```diff
@@ -0,0 +1,48 @@
+{% extends "web/pages/base.html" %}
+
+{% block title %}
+  Настройки конфиденциальности - Melomatch
+{% endblock title %}
+{% block body %}
+  {% include "web/includes/tab.html" %}
+  <div class="columns is-centered">
+    <div class="column is-three-fifths">
+      <div class="my-6">
+        <h3 class="subtitle is-3">Настройка аккаунта</h3>
+        <form method="post">
+          {% csrf_token %}
+          {% for field in form %}
+            <div class="field mt-4">
+              {% if field.name == "is_private" %}
+                <p class="has-text-weight-medium">
+                  Вы можете сделать ваш аккаунт приватным, чтобы другие пользователи не могли его просматривать.
```

Comment:
Возможно, я не совсем понятно объяснил, в чем именно заключается приватность)
```suggestion
                  Вы можете сделать ваш аккаунт приватным, чтобы другие пользователи не могли вас найти в поиске для сравнения музыкального вкуса.
```

Reply:
исправила